1_The_Physical/Guess_Point_Level_1_Stateless/3_Manage_Complexity/3_2_Boolean_Calculator/src/index.spec.ts

```diff
@@ -1,4 +1,29 @@
+import { BooleanCalculator } from "./";
 
-describe('boolean calculator', () => {
+describe(BooleanCalculator.name, () => {
```

Comment:
You've made a really good list of features and examples, why not use those to structure your test names? It would make it much easier to follow and debug